.github/workflows/continuous-integration.yaml

```diff
@@ -36,7 +36,43 @@ jobs:
         yarn build
         yarn lint
 
+    - name: 🌐 Setup local test network
+      working-directory: packages/e2e
+      run: |
+        yarn local-network:up -d
+      env:
+        POSTGRES_PORT: '5435'
+        OGMIOS_PORT: '1340'
+        OGMIOS_URL: 'ws://cardano-node-ogmios:1340'
+
     - name: 🔬 Test
       run: |
         yarn test:build:verify
         yarn test
+
+    - name: 🔬 Test - e2e
+      run: |
+        yarn workspace @cardano-sdk/e2e test:wallet
+        TX_SUBMIT_PROVIDER_PARAMS='{"baseUrl":"http://localhost:5001/tx-submit"}' yarn workspace @cardano-sdk/e2e test:cardano-services
```

Comment:
Out of curiosity, why does this change for cardano-services?

Reply:
It would be better to change the test config to use a single server if possible

Reply:
@mirceahasegan The load test inside the cardano-services starts its own version of the tx-submit service with a specific configuration, so we need to point it to that service. Maybe @iccicci can you us more context.

@rhyslbw I agree, as it is right now this will clash with the env needed by future tests added to the cardano-services e2e test suite. I think we either need to change the name of the envs on that test so it doesn't clash with other tests, or we need to somehow make it work with the tx-submit endpoint we already have, but I am not sure what would be required to do so. Do you want me to add this refactor to the scope of this PR?

Reply:
Another option would be to give it default values that would work without needing to pass the envs

Reply:
Oh, we don't need to run the load test in CI. Let's separate it out, and not run it in this context.

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/424/commits/8ecd12ac41af88dc67315efd42430158d17df340 (I am squashing since its a small change to the file overall)